src/collections/blog.item

```diff
@@ -2,93 +2,130 @@
+                            c3.536,0,6.974-1.257,9.677-3.539l131.803-111.274v176.264C343.664,295.336,340.439,297.72,336.764,297.72z M191.059,192.987
+                            L62.87,84.397h256.378L191.059,192.987z"/>
+                      </svg>
+              </a>
+              <a href="https://www.facebook.com/sharer.php?u={website.baseUrl}{fullUrl}" target="_blank" class="social-links__facebook">
+
+                      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                          <path d="M268 512V284h78.978l11.736-92H268v-62.426c0-26.352 7.665-44.31 45.455-44.31L362 85.24V3.837C353 2.728 324.7.25 291.416.25 221.926.25 174 42.666 174 120.56V192H96v92h78v228h94z"/>
+                      </svg>
+              </a>
+              <a href="https://twitter.com/share?text=Check+out+this+article+from+the+Washington+Leadership+Academy%21+" class="social-links__twitter" target="_blank">
+                      <svg class="geomicon" data-icon="twitter" viewBox="0 0 32 32">
+                          <title>twitter icon</title>
+                          <path d="M2 4 C6 8 10 12 15 11 A6 6 0 0 1 22 4 A6 6 0 0 1 26 6 A8 8 0 0 0 31 4 A8 8 0 0 1 28 8 A8 8 0 0 0 32 7 A8 8 0 0 1 28 11 A18 18 0 0 1 10 30 A18 18 0 0 1 0 27 A12 12 0 0 0 8 24 A8 8 0 0 1 3 20 A8 8 0 0 0 6 19.5 A8 8 0 0 1 0 12 A8 8 0 0 0 3 13 A8 8 0 0 1 2 4"/>
+                      </svg>
+              </a>
+          </div>
+        </div>
+        
+
+        <!--BLOG INJECTION-->
+
+        {postItemInjectCode}    
+
+        <!--CATEGORIES
+
+        {.repeated section categories}
+          <a class="category" href="{collection.fullUrl}?category={@|url-encode}">{@}</a>{.alternates with},
+        {.end}-->
+
+        <!--TAGS
+
+        {.repeated section tags}
+          <a class="tag" href="{collection.fullUrl}?tag={@|url-encode}">{@}</a>{.alternates with},
+        {.end}-->
+
+        <!--SHARE AND LIKE
+
+        {@|like-button}
+        {@|social-button}-->
+
+        <!--AUTHOR AND DATE
+        
+        <p class="meta">
+          Published on <a href="{fullUrl}"><time datetime="{addedOn|date %F}">{addedOn|date %B %d, %Y}</time></a> by <a href="{collection.fullUrl}?author={author.id}">{author.displayName}</a>.
+          {.section sourceUrl}
+            <a href="{@}" class="source" target="_blank">Source</a>.
+          {.end}
+        </p>     -->   
+
+        <!--LOCATION
+
+        {.section location}
+          {.section addressTitle}
+            {addressTitle}
+            {addressLine1}
+            {addressLine2}
+            {addressCountry}
+          {.end}
+        {.end}-->
+
+       
+
+      </article>
+    </div>
+  </div>
+  
+  
 
 {.end}
     
 
 <!--PAGINATION-->
   
 {.section pagination}
-  <nav>
+  <div class="row">
+    <nav class="news-article__pagination columns small-12 medium-10 medium-offset-1 large-8 large-offset-2">
```

Comment:
Let's try to pull `<nav class="news-article__pagination>` out of the grid markup— have it wrap around _outside_ the row, or live _inside_ the column. That will help to keep it easy to maintain and know what's going on by not overriding grid properties.